.gitignore

```diff
@@ -118,3 +118,10 @@ venv.bak/
 # Misc backups
 *.bak
 
+# Excel lockfiles
+legal-test-fixture/test/spreadsheets/.~lock*
```

Comment:
follow the patterns earlier in the ignore file. so something like
/**/.~lock*

Reply:
Resolving on new PR (rebasing is fubared)